numerrin-interface/libpcore.cpp

```diff
@@ -0,0 +1,1041 @@
+#include <sstream>
```

Comment:
Just to be clear,  my system is working (because I did add "libnumerrin4.so" in LD_LIBRARY_PATH worked and placed "numerrin.so" in the pythons module search path).  So I am all set.